hack/generated/apis/resources.md

```diff
@@ -0,0 +1,58 @@
+### microsoft.batch
+
+v1alpha1api20210101
+
+- BatchAccount
+
+### microsoft.compute
+
+v1alpha1api20200930
+
+- Disk
+
+v1alpha1api20201201
+
+- VirtualMachineScaleSet
+
+### microsoft.documentdb
+
+v1alpha1api20210515
+
+- DatabaseAccount
+
+### microsoft.managedidentity
+
+v1alpha1api20181130
+
+- UserAssignedIdentity
+
+### microsoft.network
+
+v1alpha1api20201101
```

Comment:
Consider bolding this, for emphasis?

Reply:
It's the only thing on the line, and it's followed by a bullet list, so I don't think it needs further emphasis.